osidb/signals.py

```diff
@@ -117,3 +117,14 @@ def update_local_updated_dt_tracker(sender, instance, **kwargs):
             no_alerts=True,  # recreating alerts from nested entities can cause deadlocks
             raise_validation_error=False,
         )
+
+
+@receiver(pre_save, sender=Affect)
```

Comment:
We will also need to account for the flaw change - the tracker aggregated impact is composition of its flaw impacts which may or may not be overridden by the affect impacts.